terraform/projects/app-ecs-services/config/alerts/registers.yml

```diff
@@ -9,3 +9,22 @@ groups:
     annotations:
         summary: "App {{ $labels.app }} has too many 5xx errors"
         description: "App {{ $labels.app }} has 5xx errors in excess of 25% of total requests"
+
+  - alert: Registers_DiskSpaceExceeded
+    expr: disk_utilization{org="openregister",space="prod"} > 80
```

Comment:
So here we should actually use `exported_space="prod"` rather than `space="prod"`. We cover this very briefly in https://reliability-engineering.cloudapps.digital/manuals/alerting-with-prometheus-for-paas.html#writing-your-alerting-rule-promql-expression.

The reason for this is the label for `space` actually refers to the space that your `openregister-metric-exporter` app is running in whereas `exported_space` refers to the application the metric refers to. I assume you only want to alert on apps run in your prod space (rather than those for example that are running in staging or sandbox)?

There is a technical reason behind the scenes about why this is happening and I don't think it's perfect so it may be something we try and change in the future.

Unrelated but for further interesting reading on alerting on disk space you may be interested in this article - https://www.robustperception.io/reduce-noise-from-disk-space-alerts. It may be overkill for the moment, especially as your disk usage appears to be very static but it's an interesting thing to know about.